free-programming-books.md

```diff
@@ -874,7 +874,7 @@
 * [C# Yellow Book](http://www.csharpcourse.com) (intro to programming)
 * [Creating Mobile Apps with Xamarin.Forms C#](https://developer.xamarin.com/guides/xamarin-forms/creating-mobile-apps-xamarin-forms/)
 * [Daily Design Patterns](https://www.exceptionnotfound.net/downloads/dailydesignpattern.pdf) (PDF)
-* [Data Structures and Algorithms with Object-Oriented Design Patterns in C#](http://www.brpreiss.com/books/opus6/)
+* [Data Structures and Algorithms with Object-Oriented Design Patterns in C#](http://programming.etherealspheres.com/backup/ebooks%20-%20Other%20Programming%20Languages/Data%20Structures%20And%20Algorithms%20With%20Object-Oriented%20Design%20Patterns%20In%20C%20Sharp.pdf)
```

Comment:
Thanks for tracking this down.
Is this website connected to the author? It may be that the book is no longer free as the authors site is gone- It's published by Wiley. We can use links to Internet archive wayback machine if it is there. Also, Travis wants notations when the target of a link is pdf.